source/threads.tex

```diff
@@ -7243,10 +7243,12 @@
 \end{codeblock}
 
 \begin{note}
-Line \#1 might not result in concurrency because
-the \tcode{async} call uses the default policy, which might use
-\tcode{launch::deferred}, in which case the lambda might not be invoked until the
-\tcode{get()} call; in that case, \tcode{work1} and \tcode{work2} are called on the
+It is possible for line \#1 not to result in concurrency
+because the \tcode{async} call uses the default policy,
+which possibly uses \tcode{launch::deferred},
+in which case it is possible for the lambda not to be invoked until the
```

Comment:
```suggestion
which can use \tcode{launch::deferred},
in which case it is possible that the lambda is not invoked until the
```
I think this reads a bit better. Still incredibly awkward, but maybe this is the best we can do.